Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y composer php-fpm php-cli php-gd php-xml php-apcu php-mbstring php-intl apache2 apache2-utils libfcgi0ldbl task-spooler
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN mkdir /standardebooks.org
+RUN mkdir /standardebooks.org/web
+RUN mkdir /standardebooks.org/web/config
+RUN mkdir /standardebooks.org/web/config/ssl
+RUN mkdir /var/log/local
+
+RUN openssl req -x509 -nodes -days 99999 -newkey rsa:4096 -subj "/CN=standardebooks.test" -keyout /standardebooks.org/web/config/ssl/standardebooks.test.key -sha256 -out /standardebooks.org/web/config/ssl/standardebooks.test.crt
+
+RUN a2enmod headers
+RUN a2enmod expires
+RUN a2enmod ssl
+RUN a2enmod rewrite
+RUN a2enmod proxy
+RUN a2enmod proxy_fcgi
```

Comment:
These can also all be on one line. a2enmod can enable many modules in one invocation. See the readme

Reply:
Thanks, easy. https://github.com/standardebooks/web/pull/108/commits/2b4af01b060ead9ba0624717e1460ad145cf5a3b